CalibMatCorrection.m

```diff
@@ -128,7 +128,7 @@
             end
             filteredNoOffset.(ft)=dataset.filteredFtData.(ft) -repmat(filteredOffset.(ft),size(dataset.filteredFtData.(ft),1),1);
             
-            figure,
+            figure('WindowStyle','docked'),
```

Comment:
what does wndowstyle , docked do?

Reply:
`WindowStyle`,`docked` docks the plots in a single window in the same way the `.m` files are docked in the Editor, as tabs. It was just to avoid having a millions plots to populate the desktop. This way you always have the complete list of plots on the left column of the window.